dallinger/recruiters.py

```diff
@@ -140,8 +140,12 @@ def open_recruitment(self, n=1):
             'us_only': self.config.get('us_only'),
         }
         hit_info = self.mturkservice.create_hit(**hit_request)
+        if self.config.get('launch_in_sandbox_mode'):
+            lookup_url = "https://workersandbox.mturk.com/mturk/preview?groupId={type_id}"
+        else:
+            lookup_url = "https://worker.mturk.com/mturk/preview?groupId={type_id}"
```

Comment:
I did not test this on the production environment, only in the sandbox. What do we have available?